distribution/src/bin-regular/common.sh

```diff
@@ -39,6 +39,7 @@ if [ "$JAVA_VERSION" -ge "9" ]; then
     JDK_OPTS="\
         --add-modules java.se \
         --add-exports java.base/jdk.internal.ref=ALL-UNNAMED \
+        --add-exports jdk.management/com.ibm.lang.management.internal=ALL-UNNAMED \
```

Comment:
Do we need `exports` or just open (which we use for `java.management/sun.management`) would suffice?

Reply:
Sure, `opens` would also work, based on exception message in the test i picked `exports`. This is the message:
```
java.lang.reflect.InaccessibleObjectException: Unable to make public long com.ibm.lang.management.internal.ExtendedOperatingSystemMXBeanImpl.getCommittedVirtualMemorySize() accessible: module jdk.management does not "exports com.ibm.lang.management.internal" to unnamed module
```